.github/workflows/task-checklist.yml

```diff
@@ -0,0 +1,15 @@
+name: Check all tasks are completed
+
+on:
+  check_run:
+  pull_request:
+    types: [opened, edited]
+
+jobs:
+  task-checklist:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: cd .github/actions/task-checklist && yarn install
```

Comment:
I feel like you can just put this into `.github/actions/task-checklist/action.yml`, but … it's been 2 years since I build Github Actions a lot and I don't have any examples in repos I have access to nowadays; doesn't really matter.

Reply:
had a quick look into this - doesn't look like [javascript actions](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action) support calling shell commands out of the box

[composite actions](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) seems like a potential way to make this possible, will try it out

Reply:
Yeah, that was it, composite actions…you do you though, no dramas here, it's clear what you're doing.